tests/orquestra/quantum/circuits/_gates_test.py

```diff
@@ -174,6 +174,40 @@ def test_str_on_dagger_gives_correct_representation(self):
         gate = MatrixFactoryGate("V", example_one_qubit_matrix_factory, (1, 0), 1)
         assert str(gate.dagger) == "V†(1, 0)"
 
+    def test_wrapping_two_times_gives_correct_string(self):
+        phi = sympy.Symbol("phi")
+        oper = MatrixFactoryGate("U", example_one_qubit_matrix_factory, (phi, 1), 1)
```

Comment:
[nitpick] `oper` sounds weird, I guess it's a shortcut from `operation`. I think I would do `op` as a variable name here rather than `oper`.

Reply:
Done!

Reply:
done!